yeast-core/src/helpers/diff.ts

```diff
@@ -974,12 +1069,14 @@ function diffInner(oldNodes?: YeastChild[], newNodes?: YeastChild[]): YeastChild
 						oldDiffNode.hasDiff = true;
 						oldDiffNode.diffType = DiffType.Modified;
 						oldDiffNode.diffSource = DiffSource.Old;
+						oldDiffNode.oldNodePath = (oldPath || 0) + oldIdx;
```

Comment:
The newNodePath can be added here since this is a modification. So even though 2 nodes are added to the diff document, both of those should have the same values for oldNodePath and newNodePath because they represent a modification to the same node pair.